### PR TITLE
Mention Firefox Send in FxA service list

### PR DESCRIPTION
**Pull request by timvisee**
A minor change to the FxA documentation, to list Firefox Send in the list of FxA services as well.

### CODE REVIEW

File: docs/product-portal/accounts/welcome.md [hunk 1]
```diff
@@ -225,6 +225,7 @@ Below is a list of current and future Mozilla services that delegate authenticat
 
 ### Current
 
+*   [Firefox Send](https://send.firefox.com/)
 *   [Firefox Sync](https://www.mozilla.org/en-US/firefox/sync/) (since Firefox 29)
 *   [Find My Device](https://wiki.mozilla.org/CloudServices/FindMyDevice)
 *   [Firefox Marketplace](/en-US/Marketplace)
```
Comment (vladikoff) [inline docs/product-portal/accounts/welcome.md]: *   [Find My Device](https://wiki.mozilla.org/CloudServices/FindMyDevice)
*   [Firefox Marketplace](/en-US/Marketplace)

should be deleted at some point

Comment (timvisee) [reply docs/product-portal/accounts/welcome.md]: Should I remove these two entries in this PR?

Comment (vladikoff) [reply docs/product-portal/accounts/welcome.md]: Up to you!

**Comment by vladikoff** [comment]
thank you @timvisee !

**Pull request merged**